src/omega_edit/dataEditWebView.ts

```diff
@@ -334,25 +304,17 @@ export class DataEditWebView implements vscode.Disposable {
           })
           .then(async (uri) => {
             if (uri && uri.fsPath) {
-              if (uri.path === this.fileToEdit) {
-                await this.saveSession(this.omegaSessionId, true).catch(() => {
-                  vscode.window.showErrorMessage('Failed to save')
+              await omegaEditSession
+                .saveSession(this.omegaSessionId, uri.path, true)
+                .then(async (fp) => {
+                  vscode.window.showInformationMessage(`Saved to file: ${fp}`)
```

Comment:
Concern: I'll need to load this in a real editor, but I'm not sure `saveSession` returns the file path.

Reply:
It does.  It has to, because if you set overwrite to `false`, and give it a file path that exists, a new file path will be generated in Ωedit and returned.  You can search for `save` in the C API documentation for more (see: https://ctc-oss.github.io/omega-edit/index.html).  Also testing shows this is the case.

Reply:
Sorry, bad code navigation in Github.